application/resources/js/components/Admin/Admin/Search/Search.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div>
+    <div class="form-row">
+      <SearchInputName
```

Comment:
```suggestion
      <SearchInputName :name="request.name" />
```
中身を持たないコンポーネントの場合は自己終了形式で記述しましょう
https://jp.vuejs.org/v2/style-guide/index.html#%E8%87%AA%E5%B7%B1%E7%B5%82%E4%BA%86%E5%BD%A2%E5%BC%8F%E3%81%AE%E3%82%B3%E3%83%B3%E3%83%9D%E3%83%BC%E3%83%8D%E3%83%B3%E3%83%88-%E5%BC%B7%E3%81%8F%E6%8E%A8%E5%A5%A8

またコンポーネントに指定するプロパティが１つで済む場合は改行せずに記述しましょう